src/NuGetTrends.Data/NuGetTrends.Data.csproj

```diff
@@ -2,39 +2,22 @@
 
   <PropertyGroup>
     <!--To run migrations-->
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\NuGet.Protocol.Catalog\NuGet.Protocol.Catalog.csproj" />
+    <ProjectReference Include="..\NuGet.Protocol.Catalog\NuGet.Protocol.Catalog.csproj"/>
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="2.2.4" />
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="3.1.0"/>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="3.1.0"/>
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Remove="Migrations\20180819193931_Init.cs" />
-    <Compile Remove="Migrations\20180819193931_Init.Designer.cs" />
-    <Compile Remove="Migrations\20180820172327_IndexNameVersion.cs" />
-    <Compile Remove="Migrations\20180820172327_IndexNameVersion.Designer.cs" />
-    <Compile Remove="Migrations\20180820201732_CanscadeDelete.cs" />
-    <Compile Remove="Migrations\20180820201732_CanscadeDelete.Designer.cs" />
-    <Compile Remove="Migrations\20180820210206_Index.cs" />
-    <Compile Remove="Migrations\20180820210206_Index.Designer.cs" />
-    <Compile Remove="Migrations\20180823220923_DailyDownload.cs" />
-    <Compile Remove="Migrations\20180823220923_DailyDownload.Designer.cs" />
-    <Compile Remove="Migrations\20180823221317_DailyDownload1.cs" />
-    <Compile Remove="Migrations\20180823221317_DailyDownload1.Designer.cs" />
-    <Compile Remove="Migrations\20180825172954_PendingDailyDownload.cs" />
-    <Compile Remove="Migrations\20180825172954_PendingDailyDownload.Designer.cs" />
-    <Compile Remove="Migrations\20180825230816_IndexPacakgeIdMigration.cs" />
-    <Compile Remove="Migrations\20180825230816_IndexPacakgeIdMigration.Designer.cs" />
-    <Compile Remove="Migrations\NuGetTrendsContextModelSnapshot.cs" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <Folder Include="Migrations" />
+    <Compile Remove="Migrations\20180819193931_Init.cs"/>
+    <Compile Remove="Migrations\20180819193931_Init.Designer.cs"/>
```

Comment:
These don't need to be here or?

Reply:
It was there before, I removed the other ones bcause they didn't exist on disk anymore.
Anyway I'll delete them.

Reply:
Cool.